.travis.yml

```diff
@@ -0,0 +1,7 @@
+sudo: false
```

Comment:
If we're going to add this, can you add

```
go_import_path: github.com/google/skylark
```

With that, if anyone forks + pushes from their fork to Travis CI, the tests will still work.

Reply:
Done, thank you for the suggestion